js/src/forum/addAdsenseCompat.js

```diff
@@ -0,0 +1,24 @@
+import { extend } from 'flarum/extend';
+import app from 'flarum/app';
+import Page from 'flarum/components/Page';
+
+export default function() {
+    extend(Page.prototype, 'config', function(component) {
+            if ($(".adsbygoogle").length) {
+
+                if (app.ads_active !== true) {
+                    (adsbygoogle = window.adsbygoogle || []).push({});
+                    app.ads_active = true;
+                }
+
+                if (!app.ads_index) {
```

Comment:
You are checking whether this property has been set, but are not defaulting it anywhere. Let's instantiate it above with a default value of false so that you won't need to do a conditional against undefined.

Reply:
We don't want the value assigned before hand because this code is ran on basically every new page load and we want the value preserved from the last load.